tests/test_features/test_pcoords.py

```diff
@@ -173,8 +173,12 @@ def test_parallel_coordinates_quickmethod(self):
         """
         X, y = load_occupancy(return_dataset=True).to_numpy()
 
-        #compare the images
-        visualizer = parallel_coordinates(X, y, show=False)
+        # Use only the first 100 samples so the test will run faster
+        X_t = X[:100]
+        y_t = y[:100]
+
+        # Compare the images
+        visualizer = parallel_coordinates(X_t, y_t, show=False)
```

Comment:
This is a bit 🎃 &mdash; would probably be more elegant to use the `sample` param of the quick method?

Reply:
I like this idea!  I tested this out and it works.  While I did this, I also noticed that the quick method was not passing the `shuffle` and `random_state` parameters to the visualizer.  I have made these changes on my branch.  Let me know if you would like me to push these changes.

Reply:
Good catch @pdamodaran! I'm happy to have you include those fixes with this PR or for you to open up a separate one &mdash; I suppose it might depend on how you and the other maintainers are coordinating Quick Method updates this semester, so I'll leave it to you to decide what's best!

Reply:
Thanks @rebeccabilbro!  I pushed my changes to this branch since the pcoords quick method updates had already been merged into develop.